todo_app/data/mongo_items_Vtwo.py

```diff
@@ -0,0 +1,31 @@
+from flask import Flask, render_template, request
+import pymongo
+from bson.objectid import ObjectId
+import requests
+import os
+
+def mongo_connect():
+    client = pymongo.MongoClient(os.getenv("PRIMARY_CONNECTION_STRING_DB"))
+    db = client.chaostodoSaved
+    todos = db.ToDoItems
+    
+    return todos
+
+def get_mongo_items():
+    todos = mongo_connect()
+    
+    return todos.find()
+
+def create_mongo_todo_item():
+    todos = mongo_connect()
+    
+    todos.insert_one({"name": request.form['todo-name'], "status": "To Do"})
+
+def change_mongo_status():
+    todos = mongo_connect()
+    
+    mongo_id = request.form['todo-id']
+    
+    change_status = todos.update_one({"_id": mongo_id}, {"$set":{"status": "Done"}}) 
```

Comment:
This isn't working for me, and the reason I believe is because it's not matching the right item. The underlying reason is that here you're looking for the item with id `X` but if you check the data explorer in Azure you'll see your item actually has an id `ObjectId(X)` - so Mongo is failing to match them 🙁 The solution is to bring in the ObjectId class (`from bson import ObjectId` - BSON being a library installed already as part of Pymongo) and then use it:
```suggestion
    change_status = todos.update_one({"_id": ObjectId(mongo_id)}, {"$set":{"status": "Done"}}) 
```
Edit: And I can see you already found the ObjectId class!